### PR TITLE
Import @ovh-ux/ng-ovh-doc-url

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-4506
| License          | BSD 3-Clause

## Description

Import **@ovh-ux/ng-ovh-doc-url** :truck:

Using following command:

```sh
lerna import ~/Code/github/ovh-ux/ng-ovh-doc-url \
  --dest=packages/components \
  --preserve-commit
```

### :construction_worker_man: Build

b629a89 - build: remove extra parts

### :memo: Documentation

24c5ddb - docs: update README.md file

### :bookmark: TODOs

- [x] Update CONTRIBUTORS file.

### CODE REVIEW

File: packages/components/ng-ovh-doc-url/src/template.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<a
+    target="_blank"
+    rel="noopener"
+    data-ng-href="{{:: $ctrl.url }}">
+    <span data-ng-transclude></span>
+    <i class="fa fa-external-link" aria-hidden="true"></i>
```
Comment (FredericEspiau) [inline packages/components/ng-ovh-doc-url/src/template.html]:
```suggestion
    <span class="fa fa-external-link" aria-hidden="true"></span>
    <span class="sr-only">Will open in a new tab</span>
```

File: packages/components/ng-ovh-doc-url/src/provider.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+export default function () {
+  const isValidLocaleFormat = new RegExp('^[A-Za-z]{2}_[A-Za-z]{2}');
```
Comment (FredericEspiau) [inline packages/components/ng-ovh-doc-url/src/provider.js]: Looks like a job for a `LanguageService` so we don't have language logic spread everywhere in the codebase

Comment (cbourgois) [reply packages/components/ng-ovh-doc-url/src/provider.js]: This PR target only the import of `@ovh-ux/ng-ovh-doc-url`. 
If we want to address this issue/enhancement, we should open an issue.

Comment (antleblanc) [reply packages/components/ng-ovh-doc-url/src/provider.js]: Maybe we can open an Issue to address your comments @FredericEspiau.

Let me know what do you think. Thanks!

File: packages/components/ng-ovh-doc-url/src/template.html (hunk 1 above)
Comment (antleblanc) [reply packages/components/ng-ovh-doc-url/src/template.html]: We can apply it at once when all component will be imported.

Thanks for the suggestions.

**Comment by ovh-cds** [comment]
CDS Report installation#10569.0 ✘  
* Stage 1
  * Install ✘

**Comment by ovh-cds** [comment]
CDS Report installation#10569.0 ✘  
* Stage 1
  * Install ✘

**Comment by antleblanc** [comment]
`yarn.lock` conflict resolved via the following commit: e2d5616

**Pull request merged**